addon/components/infinity-loader.js

```diff
@@ -38,14 +39,32 @@ const InfinityLoaderComponent = Ember.Component.extend({
     this.get('_scrollable').off(`${eventName}.${this.get('guid')}`);
   },
 
-  _checkIfInView() {
-    var selfOffset       = this.$().offset().top;
-    var scrollable       = this.get("_scrollable");
-    var scrollableBottom = scrollable.height() + scrollable.scrollTop();
+  _selfOffset() {
+    if(this.get('_customScrollableIsDefined')) {
+      return this.$().position().top + this.get("_scrollable").scrollTop();
+    }
+    else {
+      return this.$().offset().top;
+    }
+  },
+
+  _bottomOfScrollableOffset() {
+    return this.get("_scrollable").height() + this.get("_scrollable").scrollTop();
```

Comment:
prefer single quoted strings unless interpolating... this file isn't terribly consistent yet, but let's move in that direction